tests/test_mint.py

```diff
@@ -43,7 +43,8 @@ def create_metadata(filename: str, mint_total: int, has_targets: bool) -> str:
         metadata.append(sample)
     with open(filename, "w", newline="") as f:
         writer = csv.writer(f)
-        writer.writerows([header] + metadata)
+        writer.writerows([header])
+        writer.writerows(metadata)
```

Comment:
If separate.

```suggestion
        writer.writerow(header)
        writer.writerows(metadata)
```

Or if together, this is what i like to do.  Not that i prefer together here, but if together, i like this.

```suggestion
        writer.writerows([header, *metadata])
```